BitFaster.Caching.Benchmarks/Lfu/CmSketchNoPin.cs

```diff
@@ -267,7 +266,7 @@ private unsafe int EstimateFrequencyAvx(T value)
                     .AsUInt16();
 
                 // set the zeroed high parts of the long value to ushort.Max
-#if NET6_0
+#if NET
                 count = Avx2.Blend(count, Vector128<ushort>.AllBitsSet, 0b10101010);
 #else
                 count = Avx2.Blend(count, Vector128.Create(ushort.MaxValue), 0b10101010);
```

Comment:
This line can now be deleted - it is only required for .NET Core 3.1, because allBitsSet was introduced on .NET6.